hw/top_englishbreakfast/util/sw_sources.patch

```diff
@@ -111,9 +111,9 @@ index 0336455da..a46911701 100644
 -static dif_csrng_t csrng;
 -static dif_edn_t edn0;
 -static dif_edn_t edn1;
- 
+
  // TODO(alphan): Handle return values as long as they don't affect capture rate.
- 
```

Comment:
There seems to be some unrelated whitespace changes being made to this patch. I noticed the same thing in a previous PR by @doreis-lowrisc too. Any idea what is causing it?